<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>翻译包检查工具</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>891</width>
     <height>481</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;欢迎使用&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="execl_path_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="select_execl_btn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择execl翻译包</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cn_path_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_cn_btn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择中文翻译包</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>660</y>
     <width>891</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>清空日志</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>导出json翻译包</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>导出日志</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>select_execl_btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>select_execl_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>842</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_cn_btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>select_cn_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>905</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>971</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_execl_file()</slot>
  <slot>select_cn_file()</slot>
 </slots>
</ui>
